### DISCUSSION TITLE
why do i have to create a  discussion on your meta github repository if i want to reach you?

### DISCUSSION THREAD

**Discussion by orangedragon1029** [lishaduck/lishaduck#59] [General]
sample text

**Comment by lishaduck** [comment]
It took me a minute to figure out who you where.
Because I'm not handing out my email to spammers. Or my phone #. And I don't use instant-messaging. I prefer asynchrony.
Thus, contact me the place you already know I'm here.

P.S.: "sample text" sounds really shady 🙃

**Reply by orangedragon1029** [reply]
how did you respond so quick

**Reply by lishaduck** [reply]
I keep a gh tab open, and it said I had some notifications.

**Reply by orangedragon1029** [reply]
oh

**Reply by lishaduck** [reply]
Yep. Well, goodnight Carter.

**Reply by lishaduck** [reply]
P.S. How did _you_ respond so quick? Tell me **in person**. 🙃

**Reply by lishaduck** [reply]
Lastly, here's an article I forgot to send to you: [RSS Feeds | Open RSS](https://openrss.org/rss-feeds)
